_includes/menu.html

```diff
@@ -68,7 +68,6 @@
           {% endfor %}
         </ul>
         {% endif %}
-        {% include "searchgov/form.html" searchgov: site.searchgov %}
       </div>
```

Comment:
Since we are going to add it back, what if instead of deleting the line entirely we wrap it in a liquid comment with a "TODO", so that we can search for TODOs later and uncomment it?

Reply:
Good call, see https://github.com/18F/guides/pull/94/commits/ca572bc44dfb69df336f42d015818f8127801e12